### ISSUE TITLE
Investigate colour contrast issues within the panel component

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1132]
The panel component uses white text (`#ffffff`) on a turquoise background (`#28a197`). This provides a colour contrast of 3.2:1, which is only compliant with [WCAG 2.1 AA standards for contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) when used with large text (18pt+) or bold text greater than 14pt. Text smaller than 18pt does not meet the minimum colour contrast ratio for AA.

We could consider changing the design of the component. If we do not change the design, we should document that text below 18pt should not be used within the panel.

|     | Text | Large Text |
|-----|------|------------|
| AA  | ❌   | ✔️         |
| AAA | ❌   | ❌         |

This was flagged in an audit of the design system (5th June 2018):

> "The white /green banner content is difficult to read. The large bold H1 style display provides large letters however these bold letters provide glare which is not diffused by the light background, reading is difficult. The small text is equally problematic, the light background again fails to diffuse small amounts of glare which further decreased the clarity of text." (DAC-USER-LV-GT-T23-01)

**Comment by stevenaproctor** [comment]
@36degrees This information was in GOV.UK elements. See http://govuk-elements.herokuapp.com/colour/#examples

**Comment by joelanman** [comment]
I think for now we could just add the guidance back in, that was in Elements (see above). We just had a support request about this on Slack.

**Comment by kellylee-gds** [comment]
Closed issue as its in the done column on the Kanban board
